### PR TITLE
Add cornerRadius

### PR DESCRIPTION
**Pull request by minhaaan**
Add `cornerRadius(_ value: CGFloat)` function

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -991,13 +991,17 @@ FlexLayout also adds methods to set common UIView properties.
 
 * **`backgroundColor(_ color: UIColor)`**  
 Set the flex item's UIView background color. 
+* **`cornerRadius(_ value: CGFloat)`**  
+Set the view to rounded corner
```
Comment (lucdion) [inline README.md:995]:
```suggestion
Set the flex item's UIView rounded corner radius.
```

**Review by lucdion — APPROVED** [review]
One little change in the documentation, else its 👍 
Thanks for the addition.

File: README.md (hunk 1 above)
Comment (minhaaan) [reply README.md]: @lucdion done!

**Comment by lucdion** [comment]
Your change has been released https://github.com/layoutBox/FlexLayout/releases/tag/1.3.25
Thanks

**Comment by woohyunjin06** [comment]
@lucdion It looks like wrong tag(https://github.com/layoutBox/FlexLayout/releases/tag/1.10.1) has been uploaded with latest release.
Can you remove it?

**Comment by lucdion** [comment]
You are right, I have deleted it, thanks

**Pull request merged**